inyoka_theme_default/templates/portal/privmsg/new.html

```diff
@@ -0,0 +1,72 @@
+{#
+  portal/privmsg/new.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  This page shows a form where the user can create a new Private Message
+
+  :copyright: (c) 2013-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/privmsg/base.html' %}
+
+{% block breadcrumb %}
+  {{ super() }}
+  {{ macros.breadcrumb_item(_('Compose message')) }}
+{% endblock %}
+
+{% block content %}
+  {% call macros.outer_form(csrf_token(), form, action=href('portal', 'privmsg', 'new'),  manually_rendered=True, submit_label=_('Send message')) %}
+      {% if USER.can('send_group_pm') %}
+        {{ macros.inner_form(form, ['recipient', 'group_recipient','subject', 'text']) }}
+      {% else %}
+        {{ macros.inner_form(form, ['recipient', 'subject', 'text']) }}
+      {% endif %}
+      <input type="submit" name="preview" value="{% trans %}Preview{% endtrans %}" />
+  {% endcall %}
+  {% if preview %}
+    <div class="preview_wrapper">
+      <h2 class="title">{% trans %}Preview{% endtrans %}</h2>
+      <div id="preview" class="preview">{{ preview }}</div>
+    </div>
+  {% endif %}
+{% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+  <script type="text/javascript">
+    /* <![CDATA[ */
+    var text = new WikiEditor('textarea[name="text"]', 'forum');
+    $(function() {
```

Comment:
hm, in the ideal case we could handle the preview globally without any inline-JS (inline-JS is btw. insecure and thus terrible), so i would remove it for the moment, too